AUTHORS

```diff
@@ -19,6 +19,7 @@ Google Inc.
 JianXiong Zhou <zhoujianxiong2@gmail.com>
 Lei Xu <eddyxu@gmail.com>
 Matt Clarkson <mattyclarkson@gmail.com>
+Nick Hutchinson <nshutchinson@gmail.com>
```

Comment:
AUTHORS are people who own the copyright, ie, if a person signs the CLA on behalf of a company, that company should be in here.

CONTRIBUTORS are people who contribute but do not have copyright necessarily.

Everyone who contributes should be in CONTRIBUTORS. Everyone who claims copyright should be in AUTHORS.